### PR TITLE
Update with Experiment PHP Server SDK remote evaluation docs

### PR DESCRIPTION
**Pull request by tyiuhc**
# Amplitude Developer Docs PR

## Description

Update with Experiment PHP Server SDK remote evaluation docs

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [X] Doc update.
- [X] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [X] My documentation follows the style guidelines of this project.
- [X] I previewed my documentation on a local server using `mkdocs serve`.
- [X] Running `mkdocs serve` didn't generate any failures.
- [X] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/experiment/sdks/php-sdk.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+???config "Configuration Options"
+    | <div class="big-column">Name</div>  | Description | Default Value |
+    | --- | --- | --- |
+    | `debug` | Enable additional debug logging. | `false` |
+    | `serverUrl` | The host to fetch variants from. | `https://api.lab.amplitude.com` |
+    | `fetchTimeoutMillis` | The timeout for fetching variants in milliseconds. This timeout only applies to the initial request, not subsequent retries | `10000` |
+    | `fetchRetries` | The number of retries to attempt if a request to fetch variants fails. | `8` |
+    | `fetchRetryBackoffMinMillis` | The minimum (initial) backoff after a request to fetch variants fails. This delay is scaled by the `fetchRetryBackoffScalar` | `500` |
+    | `fetchRetryBackoffMaxMillis` | The maximum backoff between retries. If the scaled backoff becomes greater than the max, the max is used for all subsequent requests | `10000` |
```
Comment (markzegarelli) [inline docs/experiment/sdks/php-sdk.md:108]:
```suggestion
    | `fetchRetryBackoffMaxMillis` | The maximum backoff between retries. If the scaled backoff becomes greater than the maximum, Experiment uses the  maximum for all subsequent requests. | `10000` |
```

File: docs/experiment/sdks/php-sdk.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    | `fetchRetryBackoffScalar` | Scales the minimum backoff exponentially. | `1.5` |
+    | `fetchRetryTimeoutMillis` | The request timeout for retrying variant fetches. | `10000` |
+
+!!!info "EU Data Center"
+    If you're using Amplitude's EU data center, configure the `serverUrl` option on initialization to `https://api.lab.eu.amplitude.com`
```
Comment (markzegarelli) [inline docs/experiment/sdks/php-sdk.md:113]:
```suggestion
    If you use Amplitude's EU data center, set the `serverUrl` option on initialization to `https://api.lab.eu.amplitude.com`
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

**Review by markzegarelli — APPROVED** [review]

File: docs/experiment/sdks/php-sdk.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+---
+
+Official documentation for Amplitude Experiment's server-side PHP SDK implementation.
+
+![Packagist](https://img.shields.io/packagist/v/amplitude/experiment-php-server.svg)
+
+!!!info "SDK Resources"
+     [:material-github: GitHub](https://github.com/amplitude/experiment-php-server) · [:material-code-tags-check: Releases](https://github.com/amplitude/experiment-php-server/releases)
+
+!!!note
```
Comment (bgiori) [inline docs/experiment/sdks/php-sdk.md:14]:
```suggestion
```

File: docs/experiment/sdks/php-sdk.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+title: Experiment PHP SDK
+description: Official documentation for Amplitude Experiment's server-side PHP SDK implementation.
+icon: simple/php
+---
+
+Official documentation for Amplitude Experiment's server-side PHP SDK implementation.
+
+![Packagist](https://img.shields.io/packagist/v/amplitude/experiment-php-server.svg)
+
+!!!info "SDK Resources"
```
Comment (bgiori) [inline docs/experiment/sdks/php-sdk.md:11]:
```suggestion
!!!beta "SDK Resources"
```

File: docs/experiment/sdks/php-sdk.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
+---
+title: Experiment PHP SDK
```
Comment (bgiori) [inline docs/experiment/sdks/php-sdk.md:2]:
```suggestion
title: Experiment PHP SDK (Beta)
```

File: docs/experiment/sdks/php-sdk.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+You can configure the SDK client on initialization.
+
+???config "Configuration Options"
+    | <div class="big-column">Name</div>  | Description | Default Value |
+    | --- | --- | --- |
+    | `debug` | Enable additional debug logging. | `false` |
+    | `serverUrl` | The host to fetch variants from. | `https://api.lab.amplitude.com` |
+    | `fetchTimeoutMillis` | The timeout for fetching variants in milliseconds. This timeout applies to the initial request, not subsequent retries. | `10000` |
+    | `fetchRetries` | The number of retries to attempt if a request to fetch variants fails. | `8` |
+    | `fetchRetryBackoffMinMillis` | The minimum (initial) backoff after a request to fetch variants fails. This delay scales according to the the `fetchRetryBackoffScalar` setting. | `500` |
```
Comment (bgiori) [inline docs/experiment/sdks/php-sdk.md:106]:
```suggestion
    | `fetchRetryBackoffMinMillis` | The minimum (initial) backoff after a request to fetch variants fails. This delay scales according to the `fetchRetryBackoffScalar` setting. | `500` |
```

**Review by bgiori — APPROVED** [review]
🥇

**Pull request merged**